string state_machine
string IDLE = Idle
string ASCENT = Ascent
string DESCENT = Descent
string STOP = Stop

float64 time_now